daemon/aux.h

```diff
@@ -567,6 +567,7 @@ INLINE void timeval_multiply(struct timeval *result, const struct timeval *a, co
 	timeval_from_us(result, timeval_us(a) * multiplier);
 }
 INLINE void timeval_divide(struct timeval *result, const struct timeval *a, const long divisor) {
+	if (divisor == 0) return ;
```

Comment:
This is causing compiler warnings. If divisor is 0 then *result is left uninitialized which may then end up being used for other calculations further down the pipe (e.g. in timeval_totalstats_average_add())